docs/python.md

```diff
@@ -21,47 +21,360 @@ pip install dydx-python
 
 ## Example Usage
 
+### Initializing the client
+
 ```python
 from dydx.client import Client
+import dydx.constants as consts
+import dydx.util as utils
 
 # create a new client with a private key (string or bytearray)
 client = Client(
     private_key='0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d',
     node='https://parity.expotrading.com'
 )
+```
+
+### HTTP API Calls
+
+#### Trading Pairs
+
+```python
```

Comment:
Do you know why there is no syntax highlighting for python? It works for the javascript stuff. Is it this theme that you're using..?

It makes all the python code fairly hard to read

Reply:
It's not the theme, it didn't work before either